handlers/handler.go

```diff
@@ -77,19 +78,48 @@ func IndexS3Object(s3objectURL string) {
 	log.Printf("Finish to compute hashes for %s", key)
 
 	indexdInfo, _ := getIndexServiceInfo()
-	rev, err := GetIndexdRecordRev(uuid, indexdInfo.URL)
-	if err != nil {
-		log.Println(err)
-		return
+
+	var retries = 10
+	var rev = ""
+
+	for {
+		rev, err = GetIndexdRecordRev(uuid, indexdInfo.URL)
+		if err != nil {
+			retries++
+			time.Sleep(5)
```

Comment:
i'm in favor of moving the sleep call into the `if retries == MaxRetries {}` block so that on last failed retry we can break from the loop immediately, but this is minor

Reply:
We want the service taking a sleep when indexd for some reasons(overload, crash...) return non-200 status code.

Reply:
yeah i understand we want to wait and retry. I'm just saying that when we hit `retries == MaxRetries`, the code could panic immediately, rather than sleep for 5 sec and then goes to panic